File: fix_css.py
import sys

path = r'c:\xampp\htdocs\network-baba\network-baba\NB-Employer-Next\styles\allPostJobs.module.css'
with open(path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Truncate at line 1919 (0-indexed 1918)
new_lines = lines[:1919]

# Append clean definitions
footer = [
    "\n",
    "@keyframes spinAnim {\n",
    "  0% { transform: rotate(0deg); }\n",
    "  100% { transform: rotate(360deg); }\n",
    "}\n"
]

with open(path, 'w', encoding='utf-8') as f:
    f.writelines(new_lines + footer)

print("Fixed CSS file.")

File: fix_css.js
const fs = require('fs');
const path = 'c:/xampp/htdocs/network-baba/network-baba/NB-Employer-Next/styles/allPostJobs.module.css';

try {
    const data = fs.readFileSync(path, 'utf8');
    const lines = data.split(/\r?\n/);

    // Truncate at line 1919 (0-indexed 1918)
    const newLines = lines.slice(0, 1919);

    const footer = [
        "",
        "@keyframes spinAnim {",
        "  0% { transform: rotate(0deg); }",
        "  100% { transform: rotate(360deg); }",
        "}"
    ];

    fs.writeFileSync(path, newLines.join('\n') + footer.join('\n'), 'utf8');
    console.log("Fixed CSS file using Node.js.");
} catch (err) {
    console.error(err);
}
